{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Numpy | Answers"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "import autogen\n",
    "import argparse\n",
    "import sys\n",
    "import logging\n",
    "import json\n",
    "import time"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Get OpenAI API"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "gpt_model = \"gpt-3.5-turbo\"\n",
    "model_tag = \"3\"\n",
    "\n",
    "# gpt_model = \"gpt-4-turbo\"\n",
    "# model_tag = \"4\"\n",
    "\n",
    "\n",
    "llm_config = {\"model\": gpt_model,\\\n",
    "              \"api_key\": os.environ[\"OPENAI_API_KEY\"],\\\n",
    "                \"max_tokens\": 2000}"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Answer Generator Class"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "class Generator(object):\n",
    "    def __init__(self, domain, title, detail, samples):\n",
    "        self.domain = domain\n",
    "        self.title = title\n",
    "        self.detail = detail\n",
    "        self.samples = str(samples)\n",
    "        self.nitems = 10\n",
    "    \n",
    "\n",
    "    def create_writer_agent(self):\n",
    "        writer_intro = f\"\"\"\n",
    "            You are a writer acting as the interviewee. \n",
    "            Write a detailed answer to the provided question related to {self.title} in the {self.domain} sector.\n",
    "            Use the following context and key ideas to be included to get familiar with the topic: {self.detail}.\n",
    "            The answer should be in rich and informative markdown format.\n",
    "\n",
    "            Guidelines:\n",
    "            1. Answer the main question comprehensively, including math equations and code snippets if necessary.\n",
    "            2. Address each follow-up question with bullet points.\n",
    "            3. Use proper markdown format for code snippets. \n",
    "            4 In markdown subtitles use:  ###, ####, **, ***, - can be used to organize the answer.\n",
    "            5. Use $$ ... $$ for block math equations in markdown.\n",
    "            6. Use $ ... $ for inline math equations within text.\n",
    "            7. Include code snippets and math equations as needed.\n",
    "            8. Ensure that the math equations are correctly formatted in markdown.\n",
    "            9. Use bullet points or numbered lists to structure the answer.\n",
    "            10. Use emojis, bold, italic, and colored text to highlight important points.\n",
    "\n",
    "            Here is sample answer for reference: {self.samples}\n",
    "            \"\"\"\n",
    "\n",
    "\n",
    "        self.answer_writer = autogen.AssistantAgent(\n",
    "                    name=\"Writer\",\n",
    "                    description=\"Aanswer writer for provided question.\",\n",
    "                    system_message= writer_intro,\n",
    "                    llm_config=llm_config,\n",
    "                    )\n",
    "        \n",
    "    def create_reviewer_agent(self):\n",
    "        review_intro = f\"\"\"\n",
    "            You are a reviewer for the answer created by the answer writer. \\\n",
    "            You dont rewrite the answer but you will revise provided answer, make it well formatted and cleaned. \\\n",
    "            Do not add any text like reply is great, etc. as responce\\\n",
    "            You will be provided with a answer in markdown format created by the answer writer. \\\n",
    "            Your task is to review the answers and fix it if there is error and return revised one. \\\n",
    "            Make sure the answers are clear, concise and engaging. \\\n",
    "            \n",
    "            Guidelines:\n",
    "            1. Make sure the answer exists in the markdown format.\n",
    "            2. Make sure the answer is well structured with proper headings and subheadings.\n",
    "            3. Make sure code snippets and math equations are formatted properly in markdown.\n",
    "            4. Make sure Main question is highlighted with ### and Follow-up questions are highlighted with ####.\n",
    "            5. Make sure each of follow up questions highlighted with ####.\n",
    "            6. Make sure each of the follow-up questions are answered with bullet points if required.\n",
    "            7. Make sure $$ ... $$ is used for block math equations in markdown.\n",
    "            8. Make sure  $ ... $ is used for inline math equations within text.\n",
    "            9. Make sure code snippets is well formated.\n",
    "            10. Ensure that the math equations are correctly formatted in markdown.\n",
    "            11. Make sure bullet points or numbered lists is properly used to structure the answer.\n",
    "            12. Make sure emojis, bold, italic, and colored text to highlight is well formated for important points.\n",
    "\n",
    "            Here is sample answer for reference: {self.samples}\n",
    "            \"\"\"\n",
    "            \n",
    "        \n",
    "        self.answer_reviewer = autogen.AssistantAgent(\n",
    "            name=\"Reviewer\",\n",
    "            description=\"Review the answer created by the answer writer.\",\n",
    "            system_message= review_intro,\n",
    "            llm_config=llm_config)\n",
    "        \n",
    "        return\n",
    "\n",
    "\n",
    "\n",
    "    def answer_task(self):\n",
    "        task = f\"\"\"\n",
    "            Write a detailed answer to questions related to {self.title} in the {self.domain} sector.\n",
    "            Familiarize yourself with the context provided here: {self.detail}. \n",
    "            Your answer should be rich in mathematical and programmatic content, \n",
    "            presented in an informative markdown format.\n",
    "\n",
    "            Guidelines:\n",
    "            1. Main Question: Provide a comprehensive answer with adequate math equations and code snippets if necessary.\n",
    "            2. Follow-up Questions: Answer each follow-up question using bullet points.\n",
    "            3. Code Snippets: Include code snippets as needed, formatted properly in markdown.\n",
    "            4. Math Equations: Include math equations using the proper markdown format.\n",
    "            - Use $$ ... $$ for block math equations.\n",
    "            - Use $ ... $ for inline math equations within text.\n",
    "            5. String Handling: Avoid words like \"candidate's,\" \"interviewer's,\" etc., as they can disrupt the string. Use escape characters if necessary (e.g., candidate\\'s).\n",
    "\n",
    "            Formatting Tips:\n",
    "            1. Structure: Use bullet points or numbered lists to structure your answer.\n",
    "            2. Organization: Utilize subheadings to organize your content.\n",
    "            3. Clarity: Ensure sections are clear and concise.\n",
    "            4. Math in LaTeX: Format math equations in LaTeX, ensuring they render correctly in markdown.\n",
    "            5. Code Blocks: Use code blocks for any code included in your answer.\n",
    "            6. Emojis: Incorporate emojis to express ideas where appropriate.\n",
    "            7. Tables: Create tables in markdown for comparisons or structured data.\n",
    "            8. Visuals: Use images or workflow diagrams to enhance explanations.\n",
    "            9. External Links: Include links to refer to external resources for additional context.\n",
    "            10. Highlighting: Use bold and italic text to emphasize key points.\n",
    "            11. Text Color: Apply colored text to highlight important information.\n",
    "            12. Equation Appearance: Ensure math equations look good in markdown.\n",
    "            13. LaTeX Accuracy: Verify that all LaTeX parts in the markdown are correct.\n",
    "            14. Consistency: Check that subtitles and bullet points are formatted correctly.\n",
    "            Word Limit: Maximum 1000 words..\n",
    "            \"\"\"\n",
    "\n",
    "        return task\n",
    "    \n",
    "\n",
    "    def review_task(self):\n",
    "        task = f\"\"\"\n",
    "            Review the answer provided to you. It may not be in the required format. \\\n",
    "            You dont rewrite the answer but you will revise provided answer, make it well formatted and cleaned. \\\n",
    "            Do not add any text like reply is great, etc. as responce\\\n",
    "            You will be provided with a answer in markdown format created by the answer writer. \\\n",
    "            Your task is to review the answers and fix it if there is error and return revised one. \\\n",
    "            Make sure the answers are clear, concise and engaging. \\\n",
    "            \n",
    "            Guidelines:\n",
    "            1. Make sure the answer exists in the markdown format.\n",
    "            2. Make sure the answer is well structured with proper headings and subheadings.\n",
    "            3. Make sure code snippets and math equations are formatted properly in markdown.\n",
    "            4. Make sure Main question is highlighted with ### and Follow-up questions are highlighted with ####.\n",
    "            5. Make sure each of follow up questions highlighted with ####.\n",
    "            6. Make sure each of the follow-up questions are answered with bullet points if required.\n",
    "            7. Make sure $$ ... $$ is used for block math equations in markdown.\n",
    "            8. Make sure  $ ... $ is used for inline math equations within text.\n",
    "            9. Make sure code snippets is well formated.\n",
    "            10. Ensure that the math equations are correctly formatted in markdown.\n",
    "            11. Make sure bullet points or numbered lists is properly used to structure the answer.\n",
    "            12. Make sure emojis, bold, italic, and colored text to highlight is well formated for important points.\n",
    "            \"\"\"\n",
    "\n",
    "        return task\n",
    "\n",
    "    def format_question(self, question):\n",
    "        quest = \"\"\n",
    "        qtext = \"\"\n",
    "        for key, value in question.items():\n",
    "            quest += f\"{key}: {value}\\n\"\n",
    "            if key == \"Main question\":\n",
    "                qtext += f\"**{key}**: {value}\\n\\n\"\n",
    "            elif key == \"Explanation\":\n",
    "                qtext += f\"**{key}**: {value}\\n\\n\"\n",
    "            elif key == \"Follow-up questions\":\n",
    "                qtext += f\"**{key}**:\\n\\n\"\n",
    "                for i, q in enumerate(value):\n",
    "                    qtext += f\"{i+1}. {q}\\n\\n\"\n",
    "                qtext += \"\\n\\n\"\n",
    "        return quest, qtext\n",
    "    \n",
    "\n",
    "\n",
    "    def generate_answer(self, qlist):\n",
    "\n",
    "        # get answer pre-task\n",
    "        pre_task = self.answer_task()\n",
    "        #print(pre_task)\n",
    "\n",
    "        REPLY = []\n",
    "\n",
    "        for question in qlist:\n",
    "\n",
    "            quest, qtext = self.format_question(question)\n",
    "\n",
    "            task = str(pre_task) +  \"Here is the question you need to answer:\\n\"\n",
    "            task += str(quest)\n",
    "\n",
    "            #print(task)\n",
    "\n",
    "            feedback = self.answer_writer.generate_reply(messages=[{\"content\": task, \"role\": \"user\"}])\n",
    "\n",
    "            REPLY.append({\"question\":qtext, \"answer\":feedback})\n",
    "\n",
    "        return REPLY\n",
    "    \n",
    "\n",
    "    def review_answer(self, REPLY):\n",
    "\n",
    "        file_name = self.title.replace(\" \", \"_\").replace(\"-\",\"_\").lower()\n",
    "        #print(f\"Generating {self.title} answers\")\n",
    "\n",
    "        # get pre-task\n",
    "        pre_task = self.review_task()\n",
    "        #print(pre_task)\n",
    "\n",
    "        with open(\"docs/\" + file_name + \".md\", \"w\") as f:\n",
    "            for item in REPLY:\n",
    "                question = item[\"question\"]\n",
    "                answer = item[\"answer\"]\n",
    "\n",
    "                task = str(pre_task) +  \"Here is the answer you need to revise: \\n\"\n",
    "                task = task + answer\n",
    "                \n",
    "\n",
    "                feedback = self.answer_reviewer.generate_reply(messages=[{\"content\": task, \"role\": \"user\"}])\n",
    "\n",
    "                f.write(f\"## Question\\n{question}\\n\\n## Answer\\n{feedback}\\n\\n\")\n",
    "\n",
    "        print(f\"Generated {self.title} answers\")\n",
    "        print(\"-------------------------\")\n",
    "        return"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Generate Answers"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "with open(\"sectors/sectors.json\", \"r\") as f:\n",
    "    SECTORS = json.load(f)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['Introduction to Bioinformatics',\n",
       " 'Sequence Alignment',\n",
       " 'Genome Assembly',\n",
       " 'Gene Prediction',\n",
       " 'Protein Structure Prediction',\n",
       " 'Molecular Docking',\n",
       " 'Phylogenetic Analysis',\n",
       " 'Functional Genomics',\n",
       " 'Transcriptomics',\n",
       " 'Metagenomics',\n",
       " 'Introduction to Genomics',\n",
       " 'Whole Genome Sequencing',\n",
       " 'Exome Sequencing',\n",
       " 'Genome-Wide Association Studies',\n",
       " 'Comparative Genomics',\n",
       " 'Epigenomics',\n",
       " 'Structural Genomics',\n",
       " 'Functional Annotation',\n",
       " 'Introduction to Omics',\n",
       " 'Proteomics',\n",
       " 'Metabolomics',\n",
       " 'Lipidomics',\n",
       " 'Glycomics',\n",
       " 'Pharmacogenomics',\n",
       " 'Nutrigenomics',\n",
       " 'Epitranscriptomics',\n",
       " 'Single-cell Omics',\n",
       " 'Microbiomics',\n",
       " 'Interactomics',\n",
       " 'Data Integration',\n",
       " 'Data Visualization',\n",
       " 'Machine Learning in Bioinformatics',\n",
       " 'Population Genomics',\n",
       " 'Clinical Genomics',\n",
       " 'Toxicogenomics',\n",
       " 'Immunogenomics',\n",
       " 'Systems Biology',\n",
       " 'Cancer Genomics']"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "[item['titles'] for item in SECTORS]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "from examples.example_answers import examples as example_answers"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "from questions.introduction_to_bioinformatics import questions as introduction_to_bioinformatics\n",
    "from questions.sequence_alignment import questions as sequence_alignment\n",
    "from questions.genome_assembly import questions as genome_assembly\n",
    "from questions.gene_prediction import questions as gene_prediction\n",
    "from questions.protein_structure_prediction import questions as protein_structure_prediction\n",
    "from questions.molecular_docking import questions as molecular_docking\n",
    "from questions.phylogenetic_analysis import questions as phylogenetic_analysis\n",
    "from questions.functional_genomics import questions as functional_genomics\n",
    "from questions.transcriptomics import questions as transcriptomics\n",
    "from questions.metagenomics import questions as metagenomics\n",
    "from questions.introduction_to_genomics import questions as introduction_to_genomics\n",
    "from questions.whole_genome_sequencing import questions as whole_genome_sequencing\n",
    "from questions.exome_sequencing import questions as exome_sequencing\n",
    "from questions.genome_wide_association_studies import questions as genome_wide_association_studies\n",
    "from questions.comparative_genomics import questions as comparative_genomics\n",
    "from questions.epigenomics import questions as epigenomics\n",
    "from questions.structural_genomics import questions as structural_genomics\n",
    "from questions.functional_annotation import questions as functional_annotation\n",
    "from questions.introduction_to_omics import questions as introduction_to_omics\n",
    "from questions.proteomics import questions as proteomics\n",
    "from questions.metabolomics import questions as metabolomics\n",
    "from questions.lipidomics import questions as lipidomics\n",
    "from questions.glycomics import questions as glycomics\n",
    "from questions.pharmacogenomics import questions as pharmacogenomics\n",
    "from questions.nutrigenomics import questions as nutrigenomics\n",
    "from questions.epitranscriptomics import questions as epitranscriptomics\n",
    "from questions.single_cell_omics import questions as single_cell_omics\n",
    "from questions.microbiomics import questions as microbiomics\n",
    "from questions.interactomics import questions as interactomics\n",
    "from questions.data_integration import questions as data_integration\n",
    "from questions.data_visualization import questions as data_visualization\n",
    "from questions.machine_learning_in_bioinformatics import questions as machine_learning_in_bioinformatics\n",
    "from questions.population_genomics import questions as population_genomics\n",
    "from questions.clinical_genomics import questions as clinical_genomics\n",
    "from questions.toxicogenomics import questions as toxicogenomics\n",
    "from questions.immunogenomics import questions as immunogenomics\n",
    "from questions.systems_biology import questions as systems_biology\n",
    "from questions.cancer_genomics import questions as cancer_genomics"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "QLIST = { \"Introduction to Bioinformatics\": introduction_to_bioinformatics,\\\n",
    "            \"Sequence Alignment\": sequence_alignment,\\\n",
    "            \"Genome Assembly\": genome_assembly,\\\n",
    "            \"Gene Prediction\": gene_prediction,\\\n",
    "            \"Protein Structure Prediction\": protein_structure_prediction,\\\n",
    "            \"Molecular Docking\": molecular_docking,\\\n",
    "            \"Phylogenetic Analysis\": phylogenetic_analysis,\\\n",
    "            \"Functional Genomics\": functional_genomics,\\\n",
    "            \"Transcriptomics\": transcriptomics,\\\n",
    "            \"Metagenomics\": metagenomics,\\\n",
    "            \"Introduction to Genomics\": introduction_to_genomics,\\\n",
    "            \"Whole Genome Sequencing\": whole_genome_sequencing,\\\n",
    "            \"Exome Sequencing\": exome_sequencing,\\\n",
    "            \"Genome-Wide Association Studies\": genome_wide_association_studies,\\\n",
    "            \"Comparative Genomics\": comparative_genomics,\\\n",
    "            \"Epigenomics\": epigenomics,\\\n",
    "            \"Structural Genomics\": structural_genomics,\\\n",
    "            \"Functional Annotation\": functional_annotation,\\\n",
    "            \"Introduction to Omics\": introduction_to_omics,\\\n",
    "            \"Proteomics\": proteomics,\\\n",
    "            \"Metabolomics\": metabolomics,\\\n",
    "            \"Lipidomics\": lipidomics,\\\n",
    "            \"Glycomics\": glycomics,\\\n",
    "            \"Pharmacogenomics\": pharmacogenomics,\\\n",
    "            \"Nutrigenomics\": nutrigenomics,\\\n",
    "            \"Epitranscriptomics\": epitranscriptomics,\\\n",
    "            \"Single-cell Omics\": single_cell_omics,\\\n",
    "            \"Microbiomics\": microbiomics,\\\n",
    "            \"Interactomics\": interactomics,\\\n",
    "            \"Data Integration\": data_integration,\\\n",
    "            \"Data Visualization\": data_visualization,\\\n",
    "            \"Machine Learning in Bioinformatics\": machine_learning_in_bioinformatics,\\\n",
    "            \"Population Genomics\": population_genomics,\\\n",
    "            \"Clinical Genomics\": clinical_genomics,\\\n",
    "            \"Toxicogenomics\": toxicogenomics,\\\n",
    "            \"Immunogenomics\": immunogenomics,\\\n",
    "            \"Systems Biology\": systems_biology,\\\n",
    "            \"Cancer Genomics\": cancer_genomics}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Generating answers for Single-cell Omics\n",
      "Generated Single-cell Omics answers\n",
      "-------------------------\n"
     ]
    }
   ],
   "source": [
    "samples = str(example_answers)\n",
    "\n",
    "QLIST = {\n",
    "         \"Single-cell Omics\": single_cell_omics\n",
    "         }\n",
    "\n",
    "for item in SECTORS:\n",
    "    title = item[\"titles\"]\n",
    "    domain = item[\"domain\"]\n",
    "    detail = item[\"detail\"]\n",
    "    \n",
    "    if title in QLIST.keys():\n",
    "        print(f\"Generating answers for {title}\")\n",
    "\n",
    "        # get the questions\n",
    "        qlist = QLIST[title]\n",
    "\n",
    "        # create the generator\n",
    "        gen = Generator(domain, title, detail, samples)\n",
    "\n",
    "        # create the agents\n",
    "        gen.create_writer_agent()\n",
    "        gen.create_reviewer_agent()\n",
    "\n",
    "        # generate the answers\n",
    "        REPLY = gen.generate_answer(qlist)\n",
    "        \n",
    "        # review the answers\n",
    "        gen.review_answer(REPLY)\n",
    "        \n",
    "        time.sleep(5)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.7"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
